### DISCUSSION TITLE
where is the nrf51_stlink.tcl file？

### DISCUSSION THREAD

**Discussion by shang-huan** [bitcraze/discussions#465] [Q&A]
Hi. 
When I try to flash the nrf chip, it made the error as follow. 

ou@outaotaodeMacBook-Air crazyflie2-nrf-bootloader % make flash
openocd -d2 -f interface/stlink-v2.cfg  -f target/nrf51_stlink.tcl -c init -c targets -c "reset halt" \
                 -c "flash write_image erase cload_nrf.hex" -c "verify_image cload_nrf.hex" \
                 -c "mww 0x4001e504 0x01" -c "mww 0x10001080 0x3A000" -c "reset run" -c shutdown
Open On-Chip Debugger 0.11.0
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
debug_level: 2

WARNING: interface/stlink-v2.cfg is deprecated, please switch to interface/stlink.cfg
embedded:startup.tcl:26: Error: Can't find target/nrf51_stlink.tcl
in procedure 'script'
at file "embedded:startup.tcl", line 26
make: *** [flash] Error 1

It says missing files, but I don't find help in Disscussions or Documentation ...
I'm happy for every tips. Thanks.

**Comment by ataffanel** [comment]
This file is likely not part of OpenOCD anymore. If you change it for `nrf51.cfg` at the beginning of the `Makefile`, it should work.
